packages/easy-coding-standard/packages-tests/SniffRunner/ValueObject/FileTest.php

```diff
@@ -22,6 +22,6 @@ protected function setUp(): void
         $fileInfo = new SmartFileInfo(__DIR__ . '/FileSource/SomeFile.php');
 
         $this->file = $fileFactory->createFromFileInfo($fileInfo);
-        $this->file->processWithTokenListenersAndFileInfo([], $fileInfo);
+        $this->file->processWithTokenListenersAndFileInfo([], [], $fileInfo);
```

Comment:
Just to keep the original order untouched, please move the new args in the end.

Reply:
Parameters reordered